.travis.yml

```diff
@@ -26,6 +26,16 @@ sudo: false
 matrix:
   fast_finish: true
   exclude:
+  - node_js: "5"
```

Comment:
@ljharb to be clear, these are *exclusions*, so this is making node 6 the only platform in the matrix that has the example projects run.